mlflow/protos/unity_catalog_oss_service.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto2";
+
+package mlflow.unitycatalog;
+
+import "databricks.proto";
+import "scalapb/scalapb.proto";
+import "unity_catalog_oss_messages.proto";
+
+option java_package = "com.databricks.api.proto.managedcatalog";
+option java_generate_equals_and_hash = true;
+option py_generic_services = true;
+option (scalapb.options) = {
+  flat_package: true,
+};
+
+service UnityCatalogService {
+  // Registered Model Management API
+  rpc createRegisteredModel(CreateRegisteredModel) returns (CreateRegisteredModel.Response) {
+    option (rpc) = {
+      endpoints: {
+        method: "POST",
+        path: "/unity-catalog/models",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
```

Comment:
should this be `PUBLIC` instead?

Reply:
This would mean we have documentation for it somewhere. We provide this API through an abstraction to the user, they don't need to know how to use this API by themselves, so I think `PUBLIC_UNDOCUMENTED` might be more accurate here.

Reply:
we do have documentation how to use this API though:
https://docs.databricks.com/api/workspace/registeredmodels/create

Reply:
Made it public.